### PR TITLE
Feature/product cards

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds OurProducts, ProductCards components
- Adds Toggle component to MUI
- Adds hooks to provide data to product cards
- Styles many things
- Deletes demo/playground
- Deletes fixed-open-and-transparent

### CODE REVIEW

File: apps/web/src/containers/LandingPageView/components/our-products/hooks/use-product-cards.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    },
+    {
+      title: <FormattedMessage defaultMessage={'Variable Borrow'} />,
+      link: '',
+      text: (
+        <FormattedMessage
+          defaultMessage={'Choose a term and get a guaranteed interest rate.'}
+        />
+      ),
+      apy: '6.89% APY',
```
Comment (jeffywu) [inline apps/web/src/containers/LandingPageView/components/our-products/hooks/use-product-cards.tsx:95]: This should also say coming soon

File: apps/web/src/containers/LandingPageView/components/our-products/hooks/use-lend-borrow-rates.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { Market } from '@notional-finance/sdk/src/system';
+import { useAllMarkets } from '@notional-finance/notionable-hooks';
+
+export const useLendBorrowRates = () => {
+  const { cardData } = useAllMarkets();
+
+  let currentMaxRate = 0;
+  let currentMinRate: number | undefined = undefined;
+  let maxFixedLendRateData = {
```
Comment (jeffywu) [inline apps/web/src/containers/LandingPageView/components/our-products/hooks/use-lend-borrow-rates.tsx:9]: Would we rather return "undefined" here or some equivalent such that the rate does not show up at all? Not sure what will happen here during some loading state when there are no rates loaded yet.

File: apps/web/src/containers/LandingPageView/components/our-products/hooks/use-provide-liquidity-max-rate.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+interface MaxRateProvideLiquidityData {
+  symbol: string;
+  maxRate: string;
+}
+
+export const useProvideLiquidityMaxRate = (): MaxRateProvideLiquidityData => {
+  const { tradableCurrencies } = useCurrency();
+  let currentRate = 0;
+  let maxRateProvideLiquidityData = {
```
Comment (jeffywu) [inline apps/web/src/containers/LandingPageView/components/our-products/hooks/use-provide-liquidity-max-rate.tsx:13]: Similar comment to lend borrow rates, should we load a blank or spinner in the card?

File: apps/web/src/containers/LandingPageView/components/our-products/hooks/use-vault-max-rate.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+interface MaxVaultRateData {
+  symbol: string;
+  maxRate: string;
+}
+
+export const useVaultMaxRate = (): MaxVaultRateData => {
+  const { listedVaults } = useObservableState(vaultState$, initialVaultState);
+  const headlineApy = useObservableState(headlineApy$);
+  let currentRate = 0;
+  let maxVaultRateData = {
```
Comment (jeffywu) [inline apps/web/src/containers/LandingPageView/components/our-products/hooks/use-vault-max-rate.tsx:18]: Similar to other rates

File: packages/common/icons/src/assets/icons/currencies/usdc-alt.svg
Comment (jeffywu) [inline packages/common/icons/src/assets/icons/currencies/usdc-alt.svg:1]: Can we rename the "-alt" icons to "-accent-border"? It's possible we end up with more "alts" in the future.

File: apps/web/src/containers/LandingPageView/components/our-products/hooks/use-product-cards.tsx (hunk 1 above)
Comment (matthew-garrett) [reply apps/web/src/containers/LandingPageView/components/our-products/hooks/use-product-cards.tsx:95]: It does. The comingSoon variable takes care of this.

Comment (matthew-garrett) [reply apps/web/src/containers/LandingPageView/components/our-products/hooks/use-product-cards.tsx:95]: These never get rendered but I can remove them anyway

**Pull request merged**